--- xplanet-1.3.0/src/libimage/gif.c.orig	2015-12-30 22:25:37.370458740 +0000
+++ xplanet-1.3.0/src/libimage/gif.c	2015-12-30 22:25:57.250460232 +0000
@@ -179,7 +179,7 @@
 	}
     }
     
-    if (DGifCloseFile(GifFile) == GIF_ERROR) {
+    if (DGifCloseFile(GifFile,NULL) == GIF_ERROR) {
 	return(0);
     }
 
@@ -493,7 +493,7 @@
 static void QuitGifError(GifFileType *GifFile)
 {
     fprintf(stderr, "Error writing GIF file\n");
-    if (GifFile != NULL) EGifCloseFile(GifFile);
+    if (GifFile != NULL) EGifCloseFile(GifFile,NULL);
 }
 
 int 
@@ -589,7 +589,7 @@
 	Ptr += width;
     }
 
-    if (EGifCloseFile(GifFile) == GIF_ERROR)
+    if (EGifCloseFile(GifFile,NULL) == GIF_ERROR)
 
     {
 	QuitGifError(GifFile);
